fbfmaproom/fbfmaproom.py

```diff
@@ -310,7 +310,7 @@ def year_label(midpoint, season_length):
 def geometry_containing_point(
     country_key: str, point: Tuple[float, float], mode: str
 ):
-    df = retrieve_vulnerability(country_key, mode, 2020)  # arbitrary year
+    df = retrieve_shapes(country_key, mode)  # arbitrary year
```

Comment:
The comment no longer applies. Remove it.

Also you've removed all the calls to `retrieve_vulnerability`, so please remove the function itself.